apps/site/pages/es/download/index.mdx

```diff
@@ -3,6 +3,8 @@ layout: download
 title: Descarga Node.js®
 ---
 
+<WindowsWSLMessage />
```

Comment:
Also these changes can be reverted, only the `/en/` directory should be changed

Reply:
Crowdin will handle for rest of the supported languages, Right ?

Reply:
Done & Committed !